<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>980</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>LHCD_OPI_SYSTEM</string>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="statusTip" >
   <string/>
  </property>
  <property name="styleSheet" >
   <string/>
  </property>
  <property name="animated" >
   <bool>true</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QDockWidget" name="dockWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>85</height>
     </rect>
    </property>
    <property name="features" >
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas" >
     <set>Qt::NoDockWidgetArea</set>
    </property>
    <property name="windowTitle" >
     <string/>
    </property>
    <widget class="QWidget" name="dockWidgetContents" />
   </widget>
   <widget class="QGroupBox" name="groupBox_2" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>90</y>
      <width>1078</width>
      <height>860</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>QGroupBox {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color:rgb(255,255,255);
}
</string>
    </property>
    <property name="title" >
     <string/>
    </property>
    <widget class="QDockWidget" name="dockWidget_2" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1078</width>
       <height>860</height>
      </rect>
     </property>
     <property name="accessibleName" >
      <string/>
     </property>
     <property name="styleSheet" >
      <string/>
     </property>
     <property name="floating" >
      <bool>false</bool>
     </property>
     <property name="features" >
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <property name="windowTitle" >
      <string/>
     </property>
     <widget class="QWidget" name="dockWidgetContents_2" />
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>260</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>Interlock</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>410</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>LTU &amp;&amp;  DAQ</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>160</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>Klystron Power Supply</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>Operation</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>210</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>RF Controller</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>310</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>Cooling &amp; TL Gas</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>360</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font" >
     <font/>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>TC</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>910</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>QPushButton {
border: 1px solid rgb(153,153,153);
border-radius: 6px;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(204,204,204));
}
QPushButton:pressed {
border: 1px solid rgb(102,102,102);
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67, 
stop: 0 rgb(255,255,255), stop: 1 rgb(120,120,120));
}</string>
    </property>
    <property name="text" >
     <string>Mds Node</string>
    </property>
    <property name="autoDefault" >
     <bool>false</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
